astro/src/content/blog/announcing-fusionauth-1-51.mdx

```diff
@@ -0,0 +1,207 @@
+---
+publish_date: 2024-05-30
+title: Announcing FusionAuth 1.51 - Theming Toucan
+description: This update includes our new visual, Simple Theme editor and a couple bug fixes
+authors: Alex Patterson
+image: /img/blogs/release-1-51/fusionauth-1-51.png
+categories: Release Updates, Product
+tags: release notes, themes
+excerpt_separator: "{/* more */}"
+---
+import { YouTube } from '@astro-community/astro-embed-youtube';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+
+FusionAuth version 1.51.0 shipped in late May, 2024. This version includes our new visual, Simple Theme editor and a couple bug fixes.
+
+{/* more */}
+
+Something about a [Toucan Can-Can](https://www.youtube.com/shorts/5Uq0MUiQwLE), so this release shall be the "Theming Toucan".
```

Comment:
Love the video, but "Something about a Toucan Can-Can" feels like an incomplete thought.  Did you mean to add something here like Tocan's are unique and your themes should be too...".